docs/nowdns.md

```diff
@@ -0,0 +1,26 @@
+# Now-DNS
+
+## Configuration
+
+### Example
+
+```json
+{
+  "settings": [
+    {
+      "provider": "nowdns",
+      "domain": "domain.com",
+      "host": "@",
+      "username": "username",
+      "password": "password",
+    }
+  ]
+}
+```
+
+### Compulsory parameters
+
+- `"domain"` your domain name
```

Comment:
nit unnecessary description

```suggestion
- `"domain"`
```

Reply:
I felt that over documentation was better than under.